TestFoundation/TestFileManager.swift

```diff
@@ -206,6 +207,40 @@ class TestFileManager : XCTestCase {
         }
     }
     
+    func test_fileSystemAttributes() {
+        let fm = FileManager.default
+        let path = NSTemporaryDirectory()
+        
+        do {
+            let attrs = try fm.attributesOfFileSystem(forPath: path)
+            
+            XCTAssertTrue(attrs.count > 0)
+            
+            let systemNumber = attrs[.systemNumber] as? NSNumber
+            XCTAssertNotNil(systemNumber)
+            XCTAssertGreaterThan(systemNumber!.int64Value, 0)
```

Comment:
Test failures reported:

```
TestFoundation/TestFileManager.swift:221: error: TestFileManager.test_fileSystemAttributes : XCTAssertGreaterThan failed: ("-4983562779004283539") is not greater than ("0") - 
```

Given that the data is being parsed with a UInt64 in the implementation, it's not clear why this is using an Int64 in its test. Of course, if it is an UInt64 then the greaterThan 0 is basically the same as != 0, so that would be a couple of changes to fix.

Reply:
You are right I'll do